test/e2e/tests/transaction/ens.spec.ts

```diff
@@ -222,12 +88,11 @@ describe('ENS', function (this: Suite) {
       },
       async ({ driver }: { driver: Driver }) => {
         await loginWithoutBalanceValidation(driver);
-        await driver.delay(10000 * 50);
 
         // click send button on homepage to start send flow
         const homepage = new HomePage(driver);
         await homepage.check_pageIsLoaded();
-        await homepage.check_expectedBalanceIsDisplayed('<0.000001');
+        await homepage.check_expectedBalanceIsDisplayed('20');
```

Comment:
the mock adds more balance, so we are able to perform a send to ENS in the future